CHANGELOG.md

```diff
@@ -3,10 +3,11 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v1.1.0 - [date]
+## v1.1.0 - [2024-04-09]
```

Comment:
We're not releasing today, are we? I think the version should say `1.1.0dev` until we're really ready to release. There's a tool in `nf-core` to bump the version -- it's updated in 6-7 places.